### ISSUE TITLE
[Accessibility]A11y_VSCode_KeyboardShortcuts_Screenreader: Voiceover is not reading the keyboard shortcuts table.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52175]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : MAC OS HighSierra10.13.5

**Additional Details:**
Screenreader,Voiceover,MAS1.3.1

**Repro Steps:**
1. Launch VS Code while VOICEOVER is ON.
2. Navigate to Settings icon in Activity Bar->Press Enter->Select "Keyboard Shortcuts" from dropdown.
3.Using keyboard key such as "tab" navigate to "Keyboard shortcuts" table data.
4.Observed that VOICEOVER is not reading table.

**Actual:**
While reading through the screen reader such as VOICEOVER,voiceover is not reading the table as well as the provided shortcuts for reading table such as "control-option-shift-down Arrow" is not working.

**Expected:**
While reading through the voicover, voiceover should reads the table as well as provided keyboard shortcuts should be worked.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**User Impact:**
User will not able to see where the current keyboard focus is.

**MAS Reference:**
MAS1.3.1: [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_KeyboardShortcuts_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2110835/Accessibility.A11y_VSCode_KeyboardShortcuts_Screenreader.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Feels like a duplicate of https://github.com/Microsoft/vscode/issues/52154
Will rename that one
